src/types/liquidation-types.ts

```diff
@@ -65,28 +65,22 @@ export type LiquidationDashboardData = {
   vaultLiquidations: Array<VaultLiquidationsNode>;
 };
 
-export type TokenNamesResponse = {
-  vaultManagerMetrics: {
-    nodes: Array<{ id: string; liquidatingCollateralBrand: string }>;
-  };
-};
-export type LiquidationMetricsDailyNode = {
+export type VaultStateDailyNode = {
   id: string;
-  dateKey: number;
   blockTimeLast: string;
-  numActiveVaultsLast: number;
-  numLiquidatingVaultsLast: number;
-  numLiquidationsCompletedLast: number;
-  numLiquidationsAbortedLast: number;
-  liquidatingCollateralBrand: string;
+  active: number;
+  liquidated: number;
+  closed: number;
+  liquidating: number;
+  liquidatedClosed: number;
```

Comment:
I see different types in the response:

![image](https://github.com/agoric-labs/agoric-inter-dashboard/assets/60459172/9a629e6f-c8ea-4837-921b-74cf3f54769b)

I believe implicit coercion is happening and converting these strings to numbers. We should update the type for clarity and explicitly convert the values to numbers using `Number()` wherever they are used to avoid potential issues

Reply:
this implicit coercion is happening all thoughout this repo. for consistency purposes im going to keep it as is. we can handle this in a seperate PR

Reply:
I've created an issue for this which we can look later:
https://github.com/agoric-labs/agoric-inter-dashboard/issues/78

> for consistency purposes im going to keep it 

I think we should address this specific type mismatch in this PR since it has already been identified.